libgalois/include/katana/PropertyGraph.h

```diff
@@ -136,6 +149,54 @@ class KATANA_EXPORT GraphTopology {
 
   bool empty() const noexcept { return num_nodes() == 0; }
 
+  /**
+   * @param node node to get degree for
+   * @returns Degree of node N
+   */
+  size_t degree(Node node) const noexcept {
+    auto edge_range = edges(node);
+    return std::distance(edge_range.begin(), edge_range.end());
+  }
+
+  Edge original_edge_id(const Edge& eid) const noexcept { return eid; }
+
+  Node original_node_id(const Node& nid) const noexcept { return nid; }
+
+#if 0
+
+// TODO(amber): implement these if needed
+
+  void initializeLocalRanges() { }
+
+  const_local_iterator local_begin() const {
+    return const_local_iterator(this->localBegin(numNodes));
+  }
+
+  const_local_iterator local_end() const {
+    return const_local_iterator(this->localEnd(numNodes));
+  }
+
+  local_iterator local_begin() {
+    return local_iterator(this->localBegin(numNodes));
+  }
+
+  local_iterator local_end() {
+    return local_iterator(this->localEnd(numNodes));
+  }
+#endif
+
+private:
+  friend class EdgeShuffleTopology;
+  friend class EdgeLabelAwareTopology;
+
+  NUMAArray<Edge>&& MoveAdjIndices() noexcept {
```

Comment:
I would prefer that these functions are just accessors like `NUMAArray<Edge>& adj_indices()` without an additional variant for rvalues as any lvalue can be turned to an rvalue. Callers can use `std::move()` themselves.

Reply:
done.